app/_hub/kong-inc/mtls-auth/1.3-x.md

```diff
@@ -96,17 +96,31 @@ The `id` value returned can now be used for mTLS plugin configurations or consum
 
 ### Create manual mappings between certificate and Consumer object
 
-Sometimes you may not wish to use automatic Consumer lookup or you have certificates
-that contain a field value not associated with **Consumer** objects directly. In those
+Sometimes, you may not want to use automatic Consumer lookup, or you may have certificates
+that contain a field value not directly associated with **Consumer** objects. In those
 situations, you may manually assign one or more subject names to the **Consumer** object for
 identifying the correct Consumer.
 
-**Note:** "Subject names" refers to the certificate's Subject Alternative Names (SAN) or
+> **Note:** "Subject names" refers to the certificate's Subject Alternative Names (SAN) or
```

Comment:
I think italic font should be used very sparingly. It is more difficult for the reader's eye to scan.

Reply:
This is the standard blockquote styling in Markdown - we generally use it interchangeably with the blue info blocks (`div class="alert alert-ee blue"`). Maybe we should decide, as a team, which info blocks to use for notes? And then I can change the CSS for note formatting to match.

Reply:
Leaving as is for now. Will revisit note styling with team.